cypress/integration/admin-swu-score.spec.js

```diff
@@ -0,0 +1,152 @@
+/// <reference types="cypress" />
+
+describe('As an administrator authenticated via IDIR',  function() {
+    beforeEach(function() {
+        cy.sqlFixture('dbReset.sql')
+        cy.sqlFixture('users.sql')
+        cy.idirLogin('admin')
+        cy.sqlFixture('organizations.sql')
+        cy.sqlFixture('swuOpportunityPublished.sql')
+        cy.sqlFixture('swuProposals.sql')
+        cy.sqlFixture('closeAdminSWUOpportunity.sql')
+    })
+
+    it('advances proponents 1 and 2 through all evaluations and awards opportunity to proponent 1', function() {
+        cy.visit('/dashboard')
+        cy.contains('SWU created by admin').should('be.visible')
```

Comment:
Are the `be.visible`s here to reduce flakiness? if not we should be able to go straight into the `.click()` and cypress will keep retrying until it becomes visible.

Reply:
Yes, to reduce flakiness